strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NpmPackageGenerator.java

```diff
@@ -50,6 +51,11 @@ public NpmPackageGenerator(String basedir)
         packageJson.setDist(new Dist());
     }
 
+    public NpmPackageGenerator(Path basedir){
```

Comment:
Cosmetic: Please, follow the project's coding convention and keep braces on new lines. Thanks! :)